### ISSUE TITLE
Without active presentation, poll results not visible in chat

### ISSUE THREAD

**Issue by cjhille** [bigbluebutton/bigbluebutton#12141]
As requested by @antobinary in #12036

**Describe the bug**
Without an active presentation, poll results are not displayed in the chat section. Only the presenter is able to see the results before pressing the publish button. Afterwards poll results are simply gone.

**To Reproduce**
1. Start conference without presentation
2. Start poll
3. Wait for any poll results
2. Publish poll

**Expected behavior**
Poll result should be shown in chat view for all users.

**Actual behavior**
No poll result is shown in chat. Also poll UI state seems inconsistent.

**Screenshots**
![screencast 2021-04-24 02-14-39](https://user-images.githubusercontent.com/8192226/115941257-f4961380-a4a4-11eb-9a60-2c05a245bb7a.gif)

**BBB version (optional):**
BigBlueButton Server 2.3-beta-4 (dockerized version)

** Additional context:**
As soon as the presenter uploads and selects a presentation the poll results show up in chat as expected.

**Comment by joo4mart** [comment]
What do you mean with "Start conference without presentation"? Without default.pdf? 
How you have made the first step?

**Comment by cjhille** [comment]
Yes, you're right. I disabled the default presentation (for the bbb-docker deployment) via `.env`
`#DEFAULT_PRESENTATION=./conf/default_pres.pdf`
So there is no presentation uploaded in the conference. 
<img width="325" alt="VegiL Konferenz - _Test Konferenzraum 2021-04-25 14-16-05" src="https://user-images.githubusercontent.com/8192226/115993080-68cdd580-a5d1-11eb-9f34-1ddcc8a9890e.png">

This is a useful configuration in conjunction with the nextcloud plugin, that allows for individual presentation uploads upon room creation rather than a generic default presentation.

**Comment by ramonlsouza** [comment]
> Alright, thank you! This is not quite the outcome I was hoping for, but I suppose allowing a poll without an active presentation (e.g. cases when screen sharing is used primarily) is not as straight forward as implementing the error message to disallow it :)
> 
> EDIT: @ramonlsouza do you know of any github issues discussing this behaviour? Otherwise maybe a feature request for this may be in order. I don't quite understand the reasoning for disallowing polls without slides.

Yes, there are some issues requesting the same (or a similar) feature (#11936, #12237, #12103)
